elements/bulbs-video/elements/video-carousel.js

```diff
@@ -57,7 +57,10 @@ class BulbsVideoCarousel extends BulbsHTMLElement {
       '<bulbs-video-carousel> MUST contain a <bulbs-carousel>'
     );
 
-    this.videoPlayer.addEventListener('jw-beforePlay', this.firstPlay = this.firstPlay.bind(this), true);
+    // Removing for now, causing issue with videos autoplaying in background 
+    //  - bryce, 2/22/17
+    // this.videoPlayer.addEventListener('jw-beforePlay', this.firstPlay = this.firstPlay.bind(this), true);
```

Comment:
I think this is listening to the first play event to update the styling on the carousel items.